### ISSUE TITLE
How can I get number of frames?

### ISSUE THREAD

**Issue by anupzone** [ivmartel/dwv#1177]
Earlier I was using "view.getImage().getNumberOfFrames()" for getting number of frames. but in new version all changed. Can you guide me? and also for "view.getImage().getDataRange().min".

**Comment by anupzone** [comment]
also incrementFrameNb() and decrementFrameNb() changed I think. What are replacements?

**Comment by ivmartel** [comment]
Perfect! Yes quite a few api changes, I try to document them in the changelog, but I think I missed a few...

**Comment by anupzone** [comment]
Can you please guide for positionchange and wlchange event firing for display infoLayer?

**Comment by ivmartel** [comment]
The jsons of the repo should be up to date, check https://github.com/ivmartel/dwv/blob/develop/locales/en/overlays.json

**Comment by anupzone** [comment]
ok Thanks

**Comment by anupzone** [comment]
Created ios build from angular app. but during load of image giving error. attaching error and your file's code snippet. Can you guide please
<img width="996" alt="Screenshot 2022-06-21 at 5 32 07 PM" src="https://user-images.githubusercontent.com/4955675/174795027-8f37a1b2-1a71-435b-8237-4150ba6b0c40.png">
<img width="514" alt="Screenshot 2022-06-21 at 5 34 01 PM" src="https://user-images.githubusercontent.com/4955675/174795039-ec342c41-1bff-4a30-9019-43dd3c57de71.png">
?

**Comment by anupzone** [comment]
Never mind. I resolved issue. It's working fine in ios app. Problem was, cordova was over writing native web File() json. I used below method to convert:

`public blobToFile = (theBlob: Blob, fileName: string): File => {
    var b: any = theBlob;
    b.lastModifiedDate = new Date();
    b.name = fileName;
    return <File>theBlob;
  }`
![7AE90B2A-A34E-4375-8049-ADC652ACC638](https://user-images.githubusercontent.com/4955675/174822787-d50f19b6-1116-46da-be55-6c1d3ebfc52d.jpg)
